### DISCUSSION TITLE
URL & app discription

### DISCUSSION THREAD

**Discussion by dongri727** [serverpod/serverpod#1996] [Q&A] [answered]
Thanks to your help, I’ve got to deploy to AWS and publish the web app Hosting.
Thank you very much.

But still, I have a question.

Right now, the URL is app.laporte.academy,
When I look at Pixorama, it is pixorama.live and there is no ‘app'.
How can I delete the ‘app'?

![スクリーンショット 2024-03-07 13 51 28](https://github.com/serverpod/serverpod/assets/84567162/182699d0-f4c4-40bf-9f13-4ec62b8f83c2)

Also,
In the attached screen, acorn_flutter (project name as it is)
A new Flutter project(default)

Where do I rewrite it to change them ?

**Comment by SandPod** [comment, marked as answer]
Hey!

As mentioned in the [documentantion](https://docs.serverpod.dev/deployments/deploying-to-aws#what-will-be-deployed), _app.examplepod.com: The Serverpod web server. If you prefer to host it on your top domain and use www as a redirect, you can change this in the main Terraform configuration file._.

This can be configured in the terraform configuration file. (We should change the documentation so that it refers to the configuration file instead.)

If you look in `<yourappname>_server/deploy/aws/terraform/config.auto.tfvars` there is a comment here that explains what needs to be done:

```
# Subdomains for different services. Default values are recommended, but you
# may want to change the subdomain_web to www if you are using the top domain
# for the web server.
```
